### PR TITLE
Manager: use "client" instead of "daemon"

### PR DESCRIPTION
**Pull request by davidpanderson**
use uniform terminology

### CODE REVIEW

**Comment by AenBleidd** [comment]
doc/versions.inc file should not be in this pull request

**Comment by CharlieFenton** [comment]
Commit d73d512 missed changing "daemon" to "client" in these places:

* on line 629 of BOINCBaseFrame.cpp:
    strDialogTitle.Printf(
        _("%s - Daemon Start Failed"),
        pSkinAdvanced->GetApplicationName().c_str()
    );

* On lines 462 and 464 of client_state.cpp:
#ifdef _WIN32
        msg_printf(NULL, MSG_INFO, "Running as a daemon (GPU computing disabled)");
#else
        msg_printf(NULL, MSG_INFO, "Running as a daemon");
#endif

File: clientgui/BOINCGUIApp.cpp [hunk 1]
```diff
@@ -599,7 +599,7 @@ void CBOINCGUIApp::OnInitCmdLine(wxCmdLineParser &parser) {
 #if (defined(__WXMAC__) && defined(_DEBUG))
     parser.AddLongOption("NSDocumentRevisionsDebugMode", _("Not used: workaround for bug in XCode 4.2"));
 #endif
-    parser.AddSwitch("nd", "no-daemon", _("Not run the daemon"));
+    parser.AddSwitch("nd", "no-daemon", _("Don't run the client"));
```
Comment (JuhaSointusalo) [inline clientgui/BOINCGUIApp.cpp]: You could change the name of the command line switch too.

Short options are typically one character after hyphen. wxWidgets' command line parser seems to handle `-nd` without confusing it with `-n` or `-d` but I'm not sure if that's by design or by accident. I'd remove the short option.

`--no-daemon` is with hyphen but the rest of Manager and client uses underscore. Please change that.

**Comment by CharlieFenton** [comment]
Changing the command line option by eliminating or modifying either the short or the long form will break any scripts that users may have which invoke that option. It is fine to add a version with the underscore, but the old ones should remain for backward compatibility.

**Comment by JuhaSointusalo** [comment]
We have been beta testing for quite a while now so it's a bit late, but
anyone running beta release should know that anything new in it can change
or even be removed. --no-daemon is not in 7.6.33 so IMHO it can be changed.

**Pull request merged**